applications/GeoMechanicsApplication/tests/cpp_tests/test_Pw_normal_flux_condition.cpp

```diff
@@ -54,8 +54,9 @@ namespace Kratos::Testing
 
             // Create the test piping element
             std::vector<ModelPart::IndexType> cond_nodes{1, 2};
-            auto cond = r_model_part.CreateNewCondition("PwNormalFluxCondition2D2N", 1, cond_nodes, cond_prop);
-            // Initialize the element to initialize the constitutive law
+            Condition::Pointer                cond = Kratos::make_intrusive<PwNormalFluxCondition<2, 2>>(
+                1, make_shared<Line2D2<Node>>(node_1, node_2), cond_prop);
+            r_model_part.AddCondition(cond);            // Initialize the element to initialize the constitutive law
```

Comment:
Perhaps the comment should be at a separate line? Also, I believe it should refer to a condition rather than an element...?
```suggestion
            r_model_part.AddCondition(cond);
            // Initialize the condition to initialize the constitutive law
```

This suggestion also applies to one of the next changes.

Reply:
Done